recipe/meta.yaml

```diff
@@ -79,6 +82,7 @@ requirements:
     - hdf5 * {{ mpi_prefix }}_*
     - jpeg
     - libzip
+    - unzip
```

Comment:
https://github.com/conda-forge/unzip-feedstock/blob/master/recipe/meta.yaml#L14

Not available for Win :-(

Reply:
I forgot the msys version of it. Thanks!